.editorconfig

```diff
@@ -22,3 +22,6 @@ indent_size = 2
 
 [*.{yaml,yml}]
 indent_size = 2
+
+[vendor/grunt-template-jasmine-requirejs/**/*]
+indent_size = 2
```

Comment:
I probably should have done this when I vendored *grunt-template-jasmine-requirejs*. It was helpful when I had to make changes. 🤷